Documentation/deploy-master.md

```diff
@@ -668,6 +668,14 @@ kube-controller-$node
 kube-proxy-$node
 ```
 
+### Configure Calico
+
+To start Calico services :
+
+```
+docker run --rm --net=host -v /srv/kubernetes/manifests:/host/manifests quay.io/coreos/hyperkube:v1.5.1_coreos.0 /hyperkube kubectl apply -f /host/manifests/calico.yaml
```

Comment:
Hmm, other options I can think of would be:

- Running kubectl _not_ in a container by downloading the binary.
- Running the Kubernetes add-on manager.